tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2553,7 +2559,20 @@ <h2>Show similarity to selected datapoint</h2>
           self.$.dive.$.vis._onIronResize();
         });
 
-        resizer.call(dragResize);
+        // Check if the plugin was created
```

Comment:
Hmm, I realized this is in the `ready` callback instead of `attached` like the others.  Could we instead put this in an `attached` callback as well?  That way we can avoid making network requests unless the dashboard is attached to the DOM.  (I think the `resizer.call(dragResize)` line could probably stay in `ready` as it was originally.)

cc @jameswex just FYI in case you anticipate any issues here

Reply:
Unfortunately I'm not sure how to block the UI from loading / calling bad APIs without injecting it here. Can we call this good enough for now and have @jameswex or others move it with more expert knowledge?

Reply:
If this code is moved to attached, what bad APIs are called? This plugin shouldn't be making any calls in the backend until the user clicks accept in the this.$.inferencesettings dialog, which I imagine can't happen with this code, regardless of it is in atttached or ready callbacks.

Reply:
Yup - I was wrong about the code flow. Fixing.

Reply:
Looks good, thanks!  Perhaps though we should we still include the `else` clause like the other plugins?  The body would just be the initialDialog.closeDialog() call.